src/core/AutoRest/AutoRest.json

```diff
@@ -6,6 +6,12 @@
     "Azure.CSharp": {
       "type": "PluginCsa, AutoRest.CSharp.Azure"
     },
+        "Ruby": {
```

Comment:
Yeah, I saw that too. That's what I get for working on three branches concurrently that all affect the same file.

I'll make sure those are cleaned up before the promotion to `master`